client/rest-high-level/src/main/java/org/elasticsearch/client/watcher/DeactivateWatchRequest.java

```diff
@@ -28,25 +28,27 @@
     private final String watchId;
 
     public DeactivateWatchRequest(String watchId) {
+
+        if (watchId == null) {
```

Comment:
Sorry I did not explain myself well enuf. This is good, but we dont need the ValidationException here at all, really. You can instead just use `Objects.requireNotNull(obj, msg)` and it will throw exceptions for you. its more concise.

Reply:
so does that mean we don't need to check for white space in the watch id (via `PutWatchRequest.isValidId()`)?

Reply:
You should also validate this as well in the constructor.